src/node_version.h

```diff
@@ -75,44 +75,19 @@
  * Node.js will refuse to load modules that weren't compiled against its own
  * module ABI number, exposed as the process.versions.modules property.
  *
- * When this version number is changed, node.js will refuse
- * to load older modules.  This should be done whenever
- * an API is broken in the C++ side, including in v8 or
- * other dependencies.
+ * When this version number is changed, Node.js will refuse to load older
```

Comment:
nit: 'older' is not the most precise way to describe this. I would suggest replacing the with something like:

```
* Node.js will refuse to load modules with a non-matching ABI version. The
* version number here should be changed whenever an ABI-incompatible API change
* is made in the C++ side, including in V8 or other dependencies.
```

Reply:
thanks, used your wording